src/MoBi.Core/Domain/Model/MoBiProject.cs

```diff
@@ -123,6 +122,8 @@ private IReadOnlyList<T> get<T>()
 
       public IReadOnlyList<IMoBiReactionBuildingBlock> ReactionBlockCollection => get<IMoBiReactionBuildingBlock>();
 
+      public IReadOnlyList<IExpressionProfileBuildingBlock> ExpressionProfilesBlockCollection => get<IExpressionProfileBuildingBlock>();
```

Comment:
Remove the block here, I think we already have it with ExpressionProfileCollection?